Model/Config.php

```diff
@@ -334,6 +334,19 @@ public function getAllowedPaymentMethods($scope = ScopeInterface::SCOPE_STORE, $
             ? explode(',', $apm) : [];
     }
 
+    /**
+     * Get AllowedTermsPaymentMethods
+     *
+     * @param string $scope
+     * @param int $storeId
+     * @return array|string[]
+     */
+    public function getAllowedTermsPaymentMethods($scope = ScopeInterface::SCOPE_STORE, $storeId = null)
```

Comment:
How is this related to this PR?

Reply:
This is related to ME-51, I forgot to remove related changes from this PR.